packages/ses/src/module-instance.js

```diff
@@ -236,48 +312,34 @@ export const makeModuleInstance = (
     // initialized with module instances that satisfy
     // imports.
     // importedInstances = Map[_specifier_, { notifiers, module, execute }]
-    // notifiers = { _importName_: notify(update(newValue))}
+    // notifiers = { [importName: string]: notify(update(newValue))}
```

Comment:
Should we use properjs typings for these? Not in this PR but I'm interested in your reaction.

Reply:
Yes. I would like to move in the direction of tschecking jsdocs. It would be especially helpful for this module.